libs/sdk-core/makefile

```diff
@@ -17,20 +17,22 @@ init:
 	rustup target add aarch64-apple-ios x86_64-apple-ios
 	rustup target add aarch64-apple-darwin x86_64-apple-darwin
 	rustup target add aarch64-apple-ios-sim
-	#rustup target add armv7-apple-ios armv7s-apple-ios i386-apple-ios ## deprecated
 	rustup target add aarch64-linux-android armv7-linux-androideabi i686-linux-android x86_64-linux-android
+	rustup target add aarch64-unknown-linux-gnu
 	@if [ $$(uname) == "Darwin" ] ; then cargo install cargo-lipo ; fi
 	cargo install cbindgen
 	cargo install cargo-ndk
 
 ## all: Compile iOS, Android
-all: ios macos android
+all: ios macos android linux
 
 ## ios: Compile the iOS universal library
 ios: ios-universal
 
 ios-universal: $(SOURCES) ndk-home
-	cargo lipo --release
+	@if [ $$(uname) == "Darwin" ] ; then \
```

Comment:
Do we need this? Isn't it better to just fail if it can't build?

Reply:
I believe it is not allowed to build ios binaries outside of MacOS, but I'm not sure, I did add it because on Linux it fails when I tried to build it.

Reply:
Yes exactly, isn't it better that it will fail vs report DONE but won't build?

Reply:
gotcha, you are right.